### DISCUSSION TITLE
OpenUtau VST Plugin

### DISCUSSION THREAD

**Discussion by YUKI2eN3e** [stakira/OpenUtau#1105] [Ideas]
Just wanted to ask since its pointed out in the readme that "While OpenUtau can do very minimal mixing, it will not replace your digital audio workstation of choice.", would it be fisable to make a VST Plugin version of OpenUtau so you can use it in your DAW sort of like how you use Piapro Studio?

**Comment by MaximusWolfie** [comment]
a VST plugin version of OpenUTAU would be quite difficult because as far as I know, a lot of stuff would have to get, I guess, rebuilt in order to work as a VST and it just wouldn't be worth the hassle. A VST plugin would be neat though.

**Reply by YUKI2eN3e** [reply]
Yeah, that's pretty much what I figured.

**Reply by pseudo-producer** [reply]
What a shame that there is a need to rewrite the project! It would be very interesting to see Open UTAU integrating Digital Audio Stations, especially as VST/LV2 or something like that!

**Reply by pseudo-producer** [reply]
I would love to have programming knowledge to help build these ideas, but unfortunately I haven't managed to study programming languages yet...
